Dockerfile

```diff
@@ -7,6 +7,7 @@ WORKDIR /app
 COPY . .
 
 ARG TARGETOS TARGETARCH
+RUN go install github.com/swaggo/swag@latest
```

Comment:
Thank you!
In actual, i also was not sure about that!

Reply:
Thank you very much Brave! I think the command need to use `swag` command!
Instead of this, we have to add copy of api directory to `/root/api`!
@ohbyeongmin
And I'm sorry my missing about the directory of deployments. The path is `/root/deployments`!

Reply:
@nguyenzung
Oh, we are using the swag command in Dockerfile,,, I know it now. But we have to add files for swagger.

Reply:
```
FROM --platform=$BUILDPLATFORM golang:1.20-alpine AS builder

RUN apk add --no-cache make gcc musl-dev linux-headers git

WORKDIR /app

COPY . .

RUN go install github.com/swaggo/swag/cmd/swag@latest

ARG TARGETOS TARGETARCH
RUN GOOS=$TARGETOS GOARCH=$TARGETARCH make titond

FROM alpine

RUN apk add --no-cache ca-certificates jq curl
COPY --from=builder /app/api /root/api
COPY --from=builder /app/deployments /root/deployments
COPY --from=builder /app/build/bin/titond /usr/local/bin/

WORKDIR /root
ENTRYPOINT ["titond"]
```

Reply:
Thank you very much for your support
I updated and also update the base image due this issue: https://github.com/aws/aws-cli/issues/4971

Reply:
Thank you!
But I think we don't have to get aws-cli in our container! If you agree, let me change it later!

Reply:
Yes, thank you, 
I added aws-cli because I couldnot run in my laptop